app/models/user.rb

```diff
@@ -2,10 +2,11 @@ class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable, :confirmable
+  enum user_type: { candidate: 1, employer: 2, admin: 3 }
 
   # Validates
   validates :name, presence: true
-  validates :user_type, presence: true, inclusion: { in: %w(candidate employer admin) }
+  validates :user_type, presence: true, inclusion: { in: ["candidate", "employer", "admin"] }
 
-  USER_TYPE = %w(candidate employer).freeze
+  USER_TYPE = { "Candidate" => "candidate", "Employer" => "employer" }.freeze
```

Comment:
cái biến này e dùng ở những chỗ nào thế?

Reply:
cái có chữ in hoa kia là dùng ở view thôi a. con giá trị in thường là giá trị của user_type

Reply:
cái này e viết thêm để sử dụng ở view sign up của gem devise ấy a...cho người dùng chọn kiểu user_type ạ .
collection: User::USER_TYPE

Reply:
a vẫn chưa hiểu lắm cần gì phải làm ntn nhỉ :-? cụ thể chỗ nào trong view thế

Reply:
ở view/devise/registrations/new.html.erb ấy a .đoạn f.input :user_type ấy ạ..